README.md

```diff
@@ -1,18 +1,53 @@
+<!-- markdownlint-disable MD041 -->
```

Comment:
Why is this necessary? Don't we have a top-level heading in the first line?

Reply:
typo. `MD041` -> `MD014`